account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
+			"identity": model,
+			"err":      err.Error(),
+		}, "Unable to create the identity")
 		return errors.WithStack(err)
```

Comment:
@kwk Do you suggest to remove all the logs and to use the `errors.Wrap` everywhere or for this particular situation ?